src/main/java/org/opensearch/performanceanalyzer/config/setting/handler/PerformanceAnalyzerClusterSettingHandler.java

```diff
@@ -132,8 +140,20 @@ public void onSettingUpdate(final Integer newSettingValue) {
      *
      * @return the current cluster setting value if exists. Initial cluster setting otherwise.
      */
-    public int getCurrentClusterSettingValue() {
-        return currentClusterSetting;
+    public Map<String, Boolean> getCurrentClusterSettingValue() {
```

Comment:
How is the map being used by the caller now that we changed the structure of the response? Shouldn't we change that as well ?

Reply:
The result of `getCurrentClusterSettingValue()` is only used to print cluster status. It is directly passed to the `XContentBuilder` so there is no need to change anything from the caller side.

Also, we decided to keep `getCurrentClusterSettingValue()` unchanged and add a separate function `getCurrentClusterSettingValueVerbose()` so UTs, ITs are not affected.